app/views/employees/show.html.erb

```diff
@@ -3,9 +3,15 @@
 <div>Email: <%= @employee.email %></div>
 <div>Phone: <%= @employee.phone %></div>
 
+<br>
 
-<%= link_to "Edit Information", edit_employee_path(@employee) %>
-<%= link_to "Delete Employee", @employee, method: :delete, data: {confirm: "You Sure?"} %>
+<div>Office: <%= @employee.office.name %></div>
+
+<br>
+
+
+<%= link_to "Edit Information", edit_office_employee_path(@current_office, @employee) %>
+<%= link_to "Delete Employee", office_employee_path(@current_office, @employee), method: :delete, data: {confirm: "You Sure?"} %>
```

Comment:
Could you add a better confirmation message? Preferably, it should use a complete sentence 😛